core/client/fs/src/main/java/alluxio/client/block/AlluxioBlockStore.java

```diff
@@ -204,7 +204,13 @@ public BlockInStream getInStream(BlockInfo info, InStreamOptions options,
               .setLocations(locations))
           .setBlockWorkerInfos(blockWorkerInfo);
       dataSource = policy.getWorker(getWorkerOptions);
+      // TODO(lu) update the bytesReadUfs metrics
+      if (dataSource != null && mContext.hasProcessLocalWorker()
+          && dataSource.equals(mContext.getNodeLocalWorker())) {
```

Comment:
I'm not sure if this is right. Why do we consider node-local worker as process-local? Would this cause an issue somewhere when during process-local communication?

Reply:
`mContext.hasProcessLocalWorker() == true` assures the client is running together with a worker.
In this case, there is no need to go RPC to communicate with this worker

Reply:
What I get from this conversation is that people don't understand what is `PROCESS_LOCAL`, when should it happen and what does that imply. Those information should either be commented here (when `PROCESS_LOCAL` value is given) or in the enum definition (comment after each field).

Reply:
The definition is in the enum comments